www/src/components/TestingGuideline.scss

```diff
@@ -0,0 +1,5 @@
+.pgn-doc__guideline {
+    code {
+        background: #f8f8ff;
```

Comment:
Is there an SCSS color variable we could use here?

Reply:
yes, I added `$danger-100`